modules/http/http.c

```diff
@@ -173,6 +173,20 @@ http_dd_set_cipher_suite(LogDriver *d, const gchar *ciphers)
   self->ciphers = g_strdup(ciphers);
 }
 
+gboolean
+http_dd_set_tls13_cipher_suite(LogDriver *d, const gchar *tls13_ciphers)
```

Comment:
[optional]

I would prefer to use `GError` here, too, for the sake of consistency.

Reply:
I thought it's not worth adding detailed GError handling here, because the setter and the user are really close to each other and the reported error can be trivially described on the caller side, so I just followed what we currently have in the HTTP module, for example: `gboolean http_dd_set_ssl_version`.

I realized that `TLSContext` really needed those specific error descriptions because there were multiple errors that were hard to describe on the user side, especially because TLSContext is far from its user (the afsocket module).

I'd leave this as-is for now, if you don't mind, but we can of course address these later in all of our modules if we want to.